src/coreclr/src/libraries-native/entrypoints.c

```diff
@@ -19,15 +17,13 @@ typedef uint16_t UChar;
 #include "pal_idna.h"
 #include "pal_normalization.h"
 #include "pal_timeZoneInfo.h"
-#endif // TARGET_UNIX
 
-#define FCFuncStart(name) extern const void* name[]; const void* name[] = {
+#define FCFuncStart(name) EXTERN_C const void* name[]; const void* name[] = {
```

Comment:
Why do we need EXTERN_C instead of extern here? This is a C source compiled with C compiler, so all symbols should be naturally not mangled.

Reply:
When we compile from coreclr in Windows it is compiled as C++, that's why I had to add it. It seems like VC++ when it sees a C file, it expects the symbols to be extern "C" to be able to reference them from a C++ library. The linker was not able to see this symbols in Windows without doing this change.